conf/documentation.conf

```diff
@@ -1059,6 +1045,87 @@ When enabled the /status page will only be available on the
 production network. By default this is disabled
 EOT
 
+[captive_portal.httpd_mod_qos]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Enables mod_qos for the portal
+EOT
+
+[captive_portal.httpd_mod_qos_maximum_connections_per_device]
+type=numeric
+options=enabled|disabled
+description=<<EOT
+The maximum connections per device
+EOT
+
+[captive_portal.httpd_mod_evasive]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Enables mod_evasive for the portal
+EOT
+
+[captive_portal.httpd_mod_evasive_white_list]
+type=text
+description=<<EOT
+Whitelisted ip/network
+EOT
+
+[captive_portal.httpd_mod_evasive_hash_table_size]
+type=numeric
+description=<<EOT
+The hash table size
+EOT
+
+[captive_portal.httpd_mod_evasive_page_count]
+type=numeric
+description=<<EOT
+Limit the number of request on the same page
+EOT
+
+[captive_portal.httpd_mod_evasive_page_interval]
+type=numeric
+description=<<EOT
+The intervale for the page count in second
+EOT
+
+[captive_portal.httpd_mod_evasive_site_count]
+type=numeric
+description=<<EOT
+Limit the number of request on the same listener
+EOT
+
+[captive_portal.httpd_mod_evasive_site_interval]
+type=numeric
+description=<<EOT
+The intervale for the site count in second
+EOT
+
+[captive_portal.httpd_mod_evasive_blocking_period]
+type=numeric
+description=<<EOT
+The amount of time a client will be blocked
+EOT
+
+[captive_portal.httpd_mod_evasive_email_notify]
```

Comment:
That could be configurable. Since we might want to monitor actively a portal for some time then turn it off when we the situation is more stable